blockchain/chain.go

```diff
@@ -899,7 +899,7 @@ func LockTimeToSequence(isSeconds bool, locktime uint32) uint32 {
 	// Set the 22nd bit which indicates the lock time is in seconds, then
 	// shift the locktime over by 9 since the time granularity is in
 	// 512-second intervals (2^9). This results in a max lock-time of
-	// 33,554,431 seconds, or 1.06 years.
+	// 33,553,920 seconds, or 1.1 years.
```

Comment:
It's still actually 1.06 years, but it's rounded anyways, so probably doesn't matter.